dwave/system/samplers/clique.py

```diff
@@ -165,17 +165,15 @@ def sample(self, bqm, chain_strength=None, **kwargs):
 
         if chain_strength is None:
             # chain length determines chain strength
-            if embedding:
-                chain_strength = 1.5 * math.sqrt(len(embedding))
+            if embedding and bqm.num_interactions > 0:
+                squared_j = (j ** 2 for j in bqm.quadratic.values())
+                rms = math.sqrt(sum(squared_j)/bqm.num_interactions)
```

Comment:
have we checked elsewhere that `num_interactions` is positive? avoid the pesky "divide by zero"